.github/TD.js

```diff
@@ -0,0 +1,24 @@
+const num=document.querySelector(".number")

```

Comment:
I recommend changing your JavaScript code to something like this 

```
const num=document.querySelector(".number")
// const btns=document.querySelectorAll(".btns")
const coun=document.querySelector(".coun")
let count=0
function handleClick(event) {
    console.log(event)
        const display=event.target.classList
         console.log(display)
        if(display.contains('decrease')){
            count--
        }
        if(display.contains('increase')){
            count++
        }
        if(display.contains('reset')){
            count=0
        }
        num.textContent=count
    if (count>0){coun.style.color="green"}
}
```

Here I'm using the event that is passed in to determine which button has called the function and then I'm increasing, decreasing or resetting the count. This is a better approach instead of iterating through the buttons list.

Reply:
Sorry I submit the wrong file. This is not to-do app.